app/User.php

```diff
@@ -46,6 +46,14 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
+    public static function boot()
+    {
+        parent::boot();
+        self::updated(function ($user) {
```

Comment:
How hard would it be to do this only if the name is updated?

Reply:
Not hard at all, just updated this :)